core/src/components/popover/popover.scss

```diff
@@ -29,3 +29,9 @@
   overflow: auto;
   z-index: $z-index-overlay-wrapper;
 }
+
+.popover-viewport {
```

Comment:
maybe we also need to reset top/bottom here?

Reply:
what do you mean?

Reply:
so, let's say there is a ion-content or a ion-toolbar inside a popover, we don't want the top/bottom padding there, right?

it might be good to reset top/bottom to 0 too:

```
  --ion-safe-area-top: 0px;
  --ion-safe-area-bottom: 0px;
```